plutus-scb/app/Main.hs

```diff
@@ -338,68 +378,78 @@ runCliCommand minLogLevel config serviceAvailability (ForkCommands commands) =
     forkCommand ::  Command -> IO (Async ())
     forkCommand subcommand = do
       putStrLn $ "Starting: " <> show subcommand
-      asyncId <- async . void . runApp minLogLevel config . renderLogMessages . runCliCommand minLogLevel config serviceAvailability $ subcommand
+      let trace' = natTracer (\x -> TraceLoggerT $ \_ -> x) trace
```

Comment:
Not entirely sure what's going on with things like this. Maybe we need a Note about the iohk-monitoring usage?

Reply:
Added a note in https://github.com/input-output-hk/plutus/pull/2223/commits/7e97a3075ab9017813cb1abbc1da8159e71550ee